server/src/storage/segment.rs

```diff
@@ -165,15 +228,18 @@ mod tests {
     }
 
     #[test]
-    fn test_can_check_segment_is_not_full() {
-        let (data_point, mut segment_builder) = get_empty_segment_builder();
+    fn test_check_segment_is_not_full() {
+        let data_point = get_data_point();
```

Comment:
The purpose of `data_point` in this test is not clear.

Reply:
There was no purpose, it has been removed.